src/compiler/diagnosticMessages.json

```diff
@@ -2469,5 +2469,9 @@
     "A type assertion expression is not allowed in the left-hand side of an exponentiation expression. Consider enclosing the expression in parentheses.": {
       "category": "Error",
       "code": 17007
+    },
+    "The body of an 'if' statement cannot be the empty statement.": {
+      "category": "Warning",

```

Comment:
Do you mean a new concept for users? I can see `Warning` here: https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts#L2043 ...